### PR TITLE
Added Light Theme

### PR DESCRIPTION
**Pull request by DhruvThakker**
Here is the preview for light theme:
#15 #6

### CODE REVIEW

File: core.py [hunk 1]
```diff
@@ -61,6 +62,13 @@ def update(self) -> None:
...
+            self.loop = urwid.MainLoop(self.view, self.view.palette[self.theme])
+        except KeyError:
+            print ('Following are the themes available:')
+            for theme in self.view.palette.keys():
+                print (theme,) 
+            return
+
         self.update()
-        self.loop.run()
+        self.loop.run()
```
Comment (amanagr) [inline core.py]: add a new line here

File: run.py [hunk 2]
```diff
@@ -15,17 +15,20 @@ def parse_args():
...
     try:
-        ZulipController(args.config_file).main()
+        ZulipController(args.config_file, args.theme).main()
     except KeyboardInterrupt:
         print("\nThanks for using the Zulip-Terminal interface.\n")
         sys.exit(1)
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (amanagr) [inline run.py]: add a new line here

File: ui.py [hunk 3]
```diff
@@ -82,4 +92,4 @@ def main_window(self) -> Any:
         ]
         self.body = urwid.Columns(body, focus_column=1)
         w = urwid.LineBox(self.body, title=u"Zulip")
-        return w
+        return w
```
Comment (amanagr) [inline ui.py]: add a new line here

**Comment by amanagr** [comment]
Merged! @DhruvThakker

**Comment by DhruvThakker** [comment]
Thanks @amanagr

**Comment by kurianbenoy** [comment]
@DhruvThakker  , Can I add a more darker Black color?

**Pull request merged**